src/rgw/rgw_file.h

```diff
@@ -1992,7 +1992,7 @@ class RGWStatBucketRequest : public RGWLibRequest,
     return bucket.creation_time;
   }
 
-  bool only_bucket() override { return false; }
```

Comment:
IMO, we'd better keep it in a separate commit if you suspect, then if we find it harmful in the future, we could easily bisect and revert it.